apps/package.json

```diff
@@ -34,10 +35,15 @@
   "browser": {
     "blockly": "./build/package/js/blockly.js"
   },
+  "resolutions": {
+    "eshost": "github:pcardune/eshost#f3fa7fa",
+    "test262-compiler": "github:pcardune/test262-compiler#ffa0dff"
```

Comment:
Two questions:

1. It looks like the resolutions field is a yarn-specific extension of package.json.  Are we no longer backwards-compatible with npm?

2. What is customized about these repos, and do we expect to switch back to an upstream at some point?